test/io.jl

```diff
@@ -1,6 +1,7 @@
 module TestIO
     using Base.Test
     using DataFrames, Compat
+    using LaTeXStrings
```

Comment:
If this introduces a test dependency on an external package, the package will have to be included in test/REQUIRE.